### ISSUE TITLE
Replace dashes with underscores

### ISSUE THREAD

**Issue by ndporter** [LibraryCarpentry/lc-open-refine#307]
Following best practices (see [TidyData lesson](https://librarycarpentry.org/lc-spreadsheets/02-common-mistakes.html#field-name) column headers being created should use `_` instead of `-` whenever possible. This applies at least to [Looking up Data](https://librarycarpentry.org/lc-open-refine/13-looking-up-data.html) (e.g., `Journal-details`) but other episodes should be checked as well for consistency.

**Comment by ostephens** [comment]
Great suggestion @ndporter

If anyone is willing / able to check through the lesson and identify any other examples, or even better make the edits for the maintainers to then approve that would be amazing.

This [tutorial ](https://youtu.be/BpEWDg8AbPc) shows how you can show us the exact line and make the edit for maintainers to approve

**Comment by ndporter** [comment]
I won't have a chance to do this in the near future, or I would have put in a PR myself. I will also note that images may need to be changed to be consistent with the text (including the example I identified).

**Comment by jas58** [comment]
I'll add a note to also make the hyphen in User-Agent an underscore.  

And a question: Might this be an instructor note in addition to the text edits?

Also, what does the alt-text need to be?  it is currently:
Add column by fetching URLs screen capture

Does the alt-text need to describe what the image looks like (but doesn't need to repeat preceding or following instructions):
"The preview panel should read Row 1's value is 1099-4300 and the third column shows the api.crossref address completed."

**Comment by ndporter** [comment]
Good catch @bencomp it's important to only change the column headers we create, not the data we're trying to pass through the API, whose structure is determined by the providers.

**Comment by ostephens** [comment]
Confirmed that #310 only addresses the Looking up data episode, so there is still a task here to check if there are any column names that are not appropriately formatted in any other episodes

**Comment by jas58** [comment]
Is it worth adding an instructor note to that exercise for novices like myself to suggest recalling the Tidy Data principles recently taught. And at the same time pointing out the continued presence of hyphens in existing tools, such as OR's interface (popup window) or existing data. Perhaps suggest a pause to consider when instructive (manual) paragraphs may seem contradictory (as this page is).   Maybe the word "header" is metacogntively: OR will have it's own fields with headings or headers. AND the data we are editing in OR has headers or headings.  As a novice, this requires a significant pause for me.

**Comment by ostephens** [comment]
Have added #314 for the user-agent/http header issue

**Comment by ostephens** [comment]
Remaining tasks for this issue:

- [ ] Review all episodes to find any other column names suggested/used and ensure they are consistent with good practice as per the tidy data lesson (in all aspects, but including specifically using underscores not hyphens). Make sure to also take note of any screen illustrations that need updating as a result
- [ ] Update screen illustration in https://librarycarpentry.org/lc-open-refine/13-looking-up-data.html to have the underscore version of the column name rather than hyphenated
